Bencodex.Tests/Types/DictionaryTest.cs

```diff
@@ -828,6 +828,11 @@ public void HashCode()
             Assert.NotEqual(_mixedKeys.GetHashCode(), added.GetHashCode());
             Assert.Equal(added.GetHashCode(), _mixedKeys.Add("baz", "qux").GetHashCode());
             Assert.Equal(_mixedKeys.GetHashCode(), added.Remove(new Text("baz")).GetHashCode());
+
+            Assert.NotEqual(
+                Dictionary.Empty.Add("type_id", 0).Add("values", List.Empty).GetHashCode(),
+                Dictionary.Empty.Add("type_id", 1).Add("values", "foo").GetHashCode()
+            );
```

Comment:
Can we add a similar test for `List`? `List` also has the same problem. 😗

Reply:
I amended it.

https://github.com/planetarium/bencodex.net/compare/aaed7b1be19bca7b1fe7028e4438bd7133f0a2c0..1406e747adea059c08584afc7f027c3088cc0b39